mod/messages/start.php

```diff
@@ -58,7 +58,7 @@ function messages_init() {
 	elgg_register_plugin_hook_handler('container_permissions_check', 'object', 'messages_can_edit_container');
 
 	// Register actions
-	$action_path = elgg_get_plugins_path() . 'messages/actions/messages';
+	$action_path = __DIR__ . '/actions/messages';
```

Comment:
Dropping elgg_get_plugins_path for `__DIR__` is such a small change but it feels soooo much cleaner. I can see why so many 3p plugins used `elgg_get_plugins_path`...